custom/conf/app.ini.sample

```diff
@@ -493,7 +493,11 @@ SCHEDULE = @every 24h
 
 ; Clean up old repository archives
 [cron.archive_cleanup]
+; Whether to enable the job
+ENABLED = true
```

Comment:
Isn't this will change default case that would be default false before ?

Reply:
No it was true by default already. Defaults are not taken from this file but are in code

Reply:
Enabled is true by default: https://github.com/go-gitea/gitea/blob/master/modules/setting/setting.go#L425-L435

Reply:
Sorry for the mislead I was  thinking of default bool in go but I forget that we define default in settings.